scalafmt-core/shared/src/main/scala/org/scalafmt/rewrite/RewriteTrailingCommas.scala

```diff
@@ -0,0 +1,86 @@
+package org.scalafmt.rewrite
+
+import scala.meta._
+import scala.meta.classifiers.Classifier
+import scala.meta.tokens.Token
+
+import org.scalafmt.util.Whitespace
+
+case object RewriteTrailingCommas extends Rewrite {
```

Comment:
Can we rename to `RemoveTrailingCommas`? (It looks like this rule just remove trailing commas)
Or do we have any plans to add some rules that add a comma later?

Reply:
i would keep the name, regardless of which of the two options we go with (see previous comment, in `Rewrite.scala`), for two reasons:
* it's consistent with other `RewriteXXX` classes
* while it might only remove in reality, it's applied to all mutating cases (`always` and `never`).